src/murmur/PBKDF2.cpp

```diff
@@ -34,8 +34,27 @@
    SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 */
 
+#include <QtCore/QtGlobal>
+
+#ifdef Q_OS_WIN
+// <openssl/rand.h> includes <windows.h> without defining NOMINMAX,
+// which breaks our compilation because of the "max" macro.
```

Comment:
Comment says that problem exists here. But this is a workaround. It should clarify that.

Reply:
I agree.